pkgs/development/embedded/arduino/arduino-core/chrootenv.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, buildFHSUserEnv, arduino-core-unwrapped, withGui ? false, withTeensyduino ? false }:
+
+buildFHSUserEnv {
+  name = "${arduino-core-unwrapped.pname}-${arduino-core-unwrapped.version}";
+
+  targetPkgs =
+    pkgs: (with pkgs; [
+      ncurses
+      arduino-core-unwrapped
```

Comment:
currently there is a problem with the package `arduino` from "all-packages.nix":
`arduino = arduino-core.override { withGui = true; };`
it used to result into the package `arduino-1.8.16` and now results into `arduino-core-1.8.16` which could be weird to debug.

Reply:
same problem with the parameter: `withTeensyduino`